app/src/main/java/com/x8bit/bitwarden/ui/platform/theme/Type.kt

```diff
@@ -242,6 +252,17 @@ val nonMaterialTypography: NonMaterialTypography = NonMaterialTypography(
         ),
         platformStyle = PlatformTextStyle(includeFontPadding = false),
```

Comment:
I took a deeper look at the `bodySmallProminent ` & `labelMediumProminent` since those no longer exist in the new designs.

* bodySmallProminent: only exists in 1 spot and can be updated to use `bodySmall`.
* labelMediumProminent: lives in 4 places and most of them will use `bodySmall` as well, except one spot in the `CompleteRegistration` screen that will use `labelMedium`.